build.gradle

```diff
@@ -314,6 +320,12 @@ configure(project(':desktop')) {
         testAnnotationProcessor "org.projectlombok:lombok:$lombokVersion"
     }
 
+    // https://github.com/anbuck/gradle-compass
+    compass {
+        sassDir = file("src/main/resources/styles")
+        cssDir = file("build/resources/main/bisq/desktop/styles")
```

Comment:
changing that path to `cssDir = file("src/main/java/bisq/desktop/styles")` solves my problems locally. Now I can build/run it from commandline and from within IntelliJ.

Reply:
I've tested this in Linux environments, it works correctly both with the `src/` being around and without it so the CSS gets properly compiled into the jar file.  I've added this fix to the PR.